exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/SerializerTest.java

```diff
@@ -47,7 +47,7 @@ class SerializerTest {
               .setVersion("version")
               .setAttributes(Attributes.of(stringKey("ks"), "vs"))
               .build(),
-          "instrument.name",
+          "monotonic.cumulative.double.sum",
```

Comment:
All the metrics in this test used the same name, which produce name collisions with the new rules. I've changed the metric names to be a dot.delimited.version of the names of the fields they're assigned to.